var/spack/repos/builtin/packages/zee/package.py

```diff
@@ -30,8 +30,10 @@ class Zee(CMakePackage):
     variant('codechecks', default=False,
             description='Perform additional code checks like ' +
                         'formatting or static analysis')
+    depends_on('boost')
     depends_on('cmake@3:', type='build')
     depends_on('pkg-config', type='build')
+    depends_on('git', type='build', when='+codechecks')
```

Comment:
git is prerequisite with spack so I would remove it.

Reply:
I need to execute command `git merge-base --fork-point` which is not supported by `/usr/bin/git` on BB5

Reply:
Ancient RH git… :/